packages/components/src/components/modal/modal.tsx

```diff
@@ -146,8 +146,19 @@ export class Modal {
     setScrollLock(this.host, isOpen, !this.disableCloseButton && this.closeBtn, this.closeModal);
   }
 
+  private clickStartedInScrollbarTrack(e: MouseEvent): boolean {
```

Comment:
This would be a great utility function to unit test with different inputs and the expected output.